scripts/setup_test.py

```diff
@@ -514,8 +514,9 @@ def mock_isfile(path):
     def test_chrome_bin_setup_with_windows_chrome(self):
         def mock_isfile(path):
             return (
-                path == ('c:\\Program Files (x86)\\Google\\Chrome\\' +
-                         'Application\\Chrome.exe'))
+                path == (
```

Comment:
Can probably move to previous line and drop outer parens

Reply:
Done.